__tests__/integration/helpers/testRunner.js

```diff
@@ -75,7 +75,7 @@ const setUpTestRunner = async ({
   dockerComposeUpProcess.stderr.pipe(logWatcherDuplex);
 
   // uncomment this line in order to view full logs for debugging
-  // logWatcherDuplex.pipe(process.stdout);
```

Comment:
is this meant to be left uncommented or just used when debugging integration tests locally?

Reply:
It is meant to be uncommon ted while the PR is still a draft

Reply:
Removed